### PR TITLE
FSET-1659: Create Event and Sessions from UI

### PR DESCRIPTION
**Pull request by fayimora**
- Create endpoint for creating events
- Create reference endpoints for Event subtypes(FSB & Telephone Interview types)

### CODE REVIEW

File: app/controllers/EventsController.scala [hunk 1]
```diff
@@ -34,13 +34,16 @@ import scala.util.Try
 
 object EventsController extends EventsController {
   val eventsService: EventsService = EventsService
+  val eventsRepo: EventsMongoRepository = repositories.eventsRepository
```
Comment (kjharkin) [inline app/controllers/EventsController.scala]: I would say that an EventsController should not know about EventsService and EventsMongoRepository. Please refactor any direct calls on EventsMongoRepository into EventsService.

File: app/repositories/events/EventsConfigRepository.scala [hunk 2]
```diff
@@ -123,10 +138,20 @@ trait EventsConfigRepository {
       )
       eventItemFuture.recoverWith {
```
Comment (kjharkin) [inline app/repositories/events/EventsConfigRepository.scala]: Although this is always throwing an exception (if PartialFunction matches of course), I would argue that this should be ```recover``` not ```recoverWith``` - lets discuss

**Comment by kjharkin** [comment]
Wii merge after comments addressed.

**Comment by fayimora** [comment]
All done!

**Pull request merged**